traffic_ops/traffic_ops_golang/profileparameter/postparameterprofile.go

```diff
@@ -48,9 +49,28 @@ func PostParamProfile(w http.ResponseWriter, r *http.Request) {
 		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New("posting parameter profile: "+err.Error()))
 		return
 	}
+	paramName, ok, err := getParamNameFromID(*paramProfile.ParamID, inf.Tx.Tx)
+	if err != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New("getting parameter name from id: "+err.Error()))
+	} else if !ok {
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusBadRequest, nil, errors.New("parameter not found"))
```

Comment:
Shouldn't this be using `http.StatusNotFound`? Also, 'parameter not found' might be useful for the user to read, so that they know it failed because the _parameter_ wasn't found, not because the _profile_ wasn't found.